modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm

```diff
@@ -6725,6 +6759,184 @@ sub pipeline_analyses {
         },
       },
 
+      #################
+      ## PCP
+      #################
+
+      {
+        -logic_name => 'dump_fasta',
+        -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -rc_name    => '8GB',
+        -parameters => {
+          cmd => 'mkdir -p ' . $self->o('pcp_output') . ';'.
+          ' perl ' . catfile($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts', 'pcp', 'get_transcripts.pl').
+          ' -user ' . $self->o('user_r').
+          ' -dna_user ' . $self->o('user_r').
+          ' -dbname ' . $self->o('pcp_name').
+          ' -dna_dbname ' . $self->o('dna_db_name').
+          ' -host ' . $self->o('stringtie_initial_db_server').
+          ' -dna_host ' . $self->o('dna_db_server').
+          ' -port ' . $self->o('stringtie_initial_db_port').
+          ' -dna_port ' . $self->o('dna_db_port').
+          ' > ' . $self->o('pcp_output') . '/' . $self->o('pcp_name') . '.fasta',
```

Comment:
We normally use catfile/catdir to concatenate file and dir paths.

Reply:
updated to use catfile